### ISSUE TITLE
Nav Bar & Tabs - Tabindex for Inactive Elements

### ISSUE THREAD

**Issue by dakotareese** [angular/material#9383]
**Actual Behavior**:
-  `What is the issue? *` Currently the `tabindex` for inactive elements in the Nav Bar & Tab components is set to -1. This renders inactive tabs essentially invisible to screen readers.
-  `What is the expected behavior?`
  If the `tabindex` was set to zero, the same as the active tab, both the Nav Bar & Tab components would be more accessible to all users (not just screen readers).

**CodePen** (or steps to reproduce the issue): *
-  Inspect element on either the Tabs or Nav Bar demos to view the current values. If you use the tab key on your keyboard to cycle through interactive elements on the rendered page, you will see that the elements with `tabindex="-1"`are completely skipped over.

**Angular Versions**: *
-  `Angular Version:`
-  `Angular Material Version:` 1.1.0

**Notes:** If the core team would be willing to review it, my team would be happy to open a PR for this change.

![tabindex](https://cloud.githubusercontent.com/assets/1870585/17901949/e7d8f64a-692a-11e6-8908-9ac8e5444d0d.png)

**Comment by ThomasBurleson** [comment]
@dakotareese - please submit a PR and let's see if we can merge in your changes. Thank you.

**Comment by dakotareese** [comment]
Thanks @ThomasBurleson 

You'll probably be seeing more accessibility issues from me/us. I'd love to DM you somewhere to give you a little more context as to where they are coming from.

**Comment by crisbeto** [comment]
@dakotareese I'm not sure that every item is supposed to be a tab stop. At least in the `tabs` this seems to be very deliberate: the user can tab through to the active tab and then use the arrow keys navigate around, or tab again to skip directly to the content. I can imagine that this could be annoying for the user if there were 15 tabs and each one of them was a tab stop. 

That being said, I'll try to investigate why the navbar's items aren't being read out. The tabindex doesn't really help in this situation.

**Comment by crisbeto** [comment]
FYI @dakotareese here's [what the spec says regarding inactive tabs](https://www.w3.org/TR/wai-aria-practices/#tabpanel) being tab stops:

>  The active tab is placed into the tab order. Only the active tab should be in the tab order.

I'll keep the issue open to track why the navbar's tabs aren't being read out by screen readers.

**Comment by astapler** [comment]
Hi guys, I'm researching on a A11y bug on our app which utilize md-nav-bar as the global top navigation. Our bug is about "not able to use tab key to traverse through each md-nav-item link". User can us arrow key but not tab key to move to next md-nav-item due to the "tabindex=-1" on non-active links. 

I wonder if this fix here is related? Is it checked into AngularJS 1.1.x back in 2016 already? If so why the demo still has the same issue? https://material.angularjs.org/latest/demo/navBar

**Comment by Splaktar** [comment]
@astapler please see PR https://github.com/angular/material/pull/11494 for details on the current navBar a11y implementation.
